webapp/src/actions/index.ts

```diff
@@ -412,8 +412,13 @@ export function handleConnectFlow(instanceID?: string) {
 
 export function redirectConnect(instanceID: string) {
     return async (dispatch, getState) => {
+        let baseUrl = getState().entities.general.config.SiteURL;
+
+        if (baseUrl && baseUrl[baseUrl.length - 1] === '/') {
+            baseUrl = baseUrl.substr(0, baseUrl.length - 1);
+        }
         const instancePrefix = '/instance/' + btoa(instanceID);
-        const target = '/plugins/' + PluginId + instancePrefix + '/user/connect';
+        const target = baseUrl + '/plugins/' + PluginId + instancePrefix + '/user/connect';
```

Comment:
Can we use this string interpolation syntax?

```suggestion
        const target = `${baseUrl}/plugins/${PluginId}${instancePrefix}/user/connect`;
```

Reply:
@mickmister  makes sense, i'm a fan of interpolation as well.